tests/test_uwsgi_plugin.py

```diff
@@ -0,0 +1,62 @@
+from testing.sandbox import install_fake_uwsgi
+install_fake_uwsgi()
```

Comment:
It's actually really easy to spin up uwsgi.  (Internal) if you look at yelp-wsgi-encoding's tests they spin up and down uwsgi to test their behaviour.